contracts/SpokePool.sol

```diff
@@ -1242,24 +1243,10 @@ abstract contract SpokePool is
      *         INTERNAL FUNCTIONS         *
      **************************************/
 
-    function _depositV3(
-        bytes32 depositor,
-        bytes32 recipient,
-        address inputToken,
```

Comment:
Note that I’ve updated inputToken to be of type `bytes32`. It seems this was reverted to an address due to branch merges overriding the previous change to `bytes32`.

Reply:
I thought we wanted internal methods to take address types, not bytes32? Or am I misremembering?